en/stacks/tpm/index.md

```diff
@@ -0,0 +1,11 @@
+---
+title: TPM
+---
+
+# Trusted Platform Module (TPM)
+
+There are currently two different snaps that help manage TPM under Ubuntu Core Stacks
```

Comment:
We should explain what TPM is and also outline the actual difference between both snaps (one supporting just 1.2 and the other one 2.0).